absl/debugging/symbolize_elf.inc

```diff
@@ -125,7 +125,10 @@ namespace {
 // Some platforms use a special .opd section to store function pointers.
 const char kOpdSectionName[] = ".opd";
 
-#if (defined(__powerpc__) && !(_CALL_ELF > 1)) || defined(__ia64)
+// _CALL_ELF is not guaranteed to be defined by the compiler.
+#define HAVE_PPC64_ELFV2_ABI (defined(__powerpc64__) && (defined(_CALL_ELF) && (_CALL_ELF == 2)))
+
+#if HAVE_PPC64_ELFV2_ABI || defined(__ia64)
```

Comment:
/abseil-cpp/absl/debugging/symbolize_elf.inc:131:5: error: this use of "defined" may not be portable [-Werror=expansion-to-defined]